sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/SharedAccessSignature.cs

```diff
@@ -69,10 +69,10 @@ internal class SharedAccessSignature
         public string SharedAccessKey { get; private set; }
 
         /// <summary>
-        ///   The date and time that the shared access signature expires, in UTC.
+        ///   The date and time that the shared access signature expires.
         /// </summary>
         ///
-        public DateTime ExpirationUtc { get; private set; }
+        public DateTimeOffset ExpirationTime { get; private set; }
```

Comment:
Can we go with `SignatureExpiration` or similar here?

Reply:
Renamed it to `SignatureExpiration`.